src/Components/PlaylistCard.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { useData } from "../Context/DataProvider";
+import {
+  CheckIfVideoExistsInList,
+  formatDate,
+  formatNumber,
+  formatString
+} from "../utils";
+import { Link } from "react-router-dom";
+import { HiDotsVertical } from "react-icons/hi";
+import { MdWatchLater, MdRemoveCircle } from "react-icons/md";
+export function PlaylistCard({ videoId, playlistId }) {
+  const {
+    state: { videos, watchLater },
+    addToHistory,
+    addToWatchLater,
+    dispatch
+  } = useData();
+  function getPlaylist() {
+    return videos.find((video) => video.id === videoId);
+  }
+  const video = getPlaylist();
+  function removeFromPlaylist() {
+    dispatch({
+      type: "SET_VIDEO_ID_TO_BE_ADDED_TO_PLAYLIST",
```

Comment:
Please take a look at this constant name, you can rename it as ADD_VIDEO_TO_PLAYLIST.

Reply:
I cannot rename it to ADD_VIDEO_TO_PLAYLIST because here iam just setting variable state which iam using in comparison while adding video to playlist.